service/src/test/java/greencity/filters/RatingStatisticsSpecificationTest.java

```diff
@@ -1,17 +1,22 @@
 package greencity.filters;
 
+import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.mockito.Mockito.*;
+
 import greencity.enums.RatingCalculationEnum;
 import greencity.dto.ratingstatistics.RatingStatisticsViewDto;
 import greencity.entity.RatingStatistics;
 import greencity.entity.RatingStatistics_;
 import greencity.entity.User;
 import greencity.entity.User_;
+
 import java.time.ZonedDateTime;
 import java.util.ArrayList;
 import java.util.List;
 import javax.persistence.criteria.*;
 import javax.persistence.metamodel.SingularAttribute;
+
+import greencity.exception.exceptions.NotFoundException;
```

Comment:
please move this import to another `greencity` imports

Reply:
fixed